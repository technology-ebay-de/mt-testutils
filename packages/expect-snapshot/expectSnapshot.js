/* global expect */

import renderer from 'react-test-renderer';

export default component => expect(renderer.create(component).toJSON()).toMatchSnapshot();
