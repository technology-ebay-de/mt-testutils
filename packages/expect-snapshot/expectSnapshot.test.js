import React from 'react';
import expectSnapshot from './expectSnapshot';

const mockComponent = <div>test</div>;

describe('[app/modules/testing/expectRenderedToMatchSnapshot]', () => {
    it('renders correctly', () => expectSnapshot(mockComponent));
});
