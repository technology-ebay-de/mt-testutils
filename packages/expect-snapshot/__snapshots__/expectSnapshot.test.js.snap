// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[app/modules/testing/expectRenderedToMatchSnapshot] renders correctly 1`] = `
<div>
  test
</div>
`;
